resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache

```diff
@@ -261,9 +261,8 @@ public class WebMessagingClient {
         try {
             GenerateUploadUrlRequest generateUploadUrlRequest = new GenerateUploadUrlRequest()
               .token(this.token)
-              .action(RequestTypeGenerateUploadUrl.ONGENERATEUPLOADURL)
+							.action(RequestTypeGenerateUploadUrl.ONATTACHMENT)
```

Comment:
I don't think you should change the action

Reply:
this is needed to be in sync with the swagger file updated. I guess you have the same in your branch